### ISSUE TITLE
Contributor has signed CLA but sill shows pending CLA on PR for certain branch every time

### ISSUE THREAD

**Issue by jcwchen** [cla-assistant/cla-assistant#840]
## URL to the linked Repository or Organization

https://github.com/onnx/onnx
## Steps to reproduce the problem

If a developer (who has already signed CLA) makes a PR in onnx/onnx targeting certain branch, the CLA check in that PR will be always pending. Besides, basically it happened every time for that branch... The only workaround now is using the command (`https://cla-assistant.io/check/<orgname>/<reponame>?pullRequest=<pr_number>`), but it is quite troublesome.

## What is the expected result?
CLA should show signed.

## What happens instead?
CLA is always pending even though the contributor has signed CLA.

## Any other information? (attach screenshot if possible)
An interesting finding is actually it mostly happened to PRs targeting certain branch (e.g., rel-1.11.0). For main branch, typically CLA works pretty well. So I was wondering perhaps I miss some settings? However, I cannot find a solution other than always manually checking CLA by the link above. How can I solve this issue? Thank you for your assistance!

## Used browser and browser version?
Chrome.

**Comment by james-deee** [comment]
Yeah, I am having this same issue on this PR https://github.com/grafana/grafana/pull/47290

**Comment by KharitonOff** [comment]
@vitomadio the solution to your issue is described here https://github.com/cla-assistant/cla-assistant/issues/562#issuecomment-619914668

**Comment by vitomadio** [comment]
> @vitomadio the solution to your issue is described here [#562 (comment)](https://github.com/cla-assistant/cla-assistant/issues/562#issuecomment-619914668)

Yes mate, that was indeed the issue, thx.
